definitions/subworkflows/cellranger_mkfastq_and_count.cwl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "running cellranger mkfastq and count"
+
+inputs:
+    bcl_directory:
+        type: string
+    chemistry:
+        type: string?
+    reference:
+        type: string
+    sample_name:
+        type: string
+    simple_sample_csv:
+        type: string
```

Comment:
Looks like we're going all-in on the "strings for filepaths" approach in this workflow.